File: imobil-project/bin/Persoana.dart
class Persoana
{
  String nume;
  String email;
  int varsta;
  int telefon;

  Persoana(String nume, int varsta, int telefon, String email)
  {

    this.nume = nume;
    this.varsta = varsta;
    this.telefon = telefon;
    this.email = email;
  }
  String get Nume => nume;
  set NUME(String value) => nume=value;

  String get Email => email;
  set Email(String value) => email=value;

  int get Varsta => varsta;
  set Varsta(int value) => varsta=value;

  int get Telefon => telefon;
  set Telefon(int value) => telefon=value;
@override
  String toString() {
  StringBuffer sb = new StringBuffer();
  sb.writeln('*');
  sb.writeln('NUME: ' + nume + ', ');
  sb.writeln('VARSTA: ' + varsta.toString() + ', ');
  sb.writeln('NUMAR DE TELEFON: ' + telefon.toString() + ', ');
  sb.writeln('EMAIL: ' + email+', ');
  sb.writeln('* ');
  return sb.toString();

  }
}

File: Imobil C#/Imobil/Persoana.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Imobil
{
    public class Persoana
    {
        public string nume;
        public string email;
        public int varsta;
        public long telefon;
        
        public Persoana(string nume, int varsta, long telefon, string email)
        {
           
            this.nume = nume;
            this.varsta = varsta;
            this.telefon = telefon;
            this.email = email;
        }
        public string Nume
        {
            get
            {
                return nume;
            }

            set
            {
                nume = value;
            }


        }
      
        public int Varsta
        {
            get
            {
                return varsta;

            }
            set
            {
                varsta = value;
            }
        }
        public string Email
        {
            get
            {
                return email;
            }
            set
            {
                email = value;
            }

        }
        public long Telefon
        {
            get
            {
                return telefon;
            }
            set
            {
                telefon = value;
            }
        }
        
        public override string ToString()
        {
            StringBuilder sb = new StringBuilder();
            sb.Append("*");
            sb.Append("NUME: " + nume + ", ");
            sb.Append("VARSTA: " + varsta + ", ");
            sb.Append("NUMAR DE TELEFON: " + telefon + ", ");
            sb.Append("EMAIL: " + email+", ");
            sb.Append(" * ");
            return sb.ToString();
        }
    
    }
}
